bridge-ffi/src/lib.rs

```diff
@@ -500,27 +952,19 @@ impl tmprl_core_t {
                     .map_err(|err| format!("invalid telemetry options: {}", err))?,
             );
         }
-        let core_opts = core_opts
+        core_opts
             .build()
-            .map_err(|err| format!("invalid options: {}", err))?;
-        Ok(tmprl_core_t {
-            tokio_runtime,
-            core: std::sync::Arc::new(
-                temporal_sdk_core::init(core_opts)
-                    .await
-                    .map_err(|err| format!("failed initializing: {}", err))?,
-            ),
-        })
+            .map_err(|err| format!("invalid options: {}", err))
     }
+}
 
-    async fn shutdown(&self) {
-        self.core.shutdown().await;
-    }
+// Present for try-from only
+struct WorkerConfig(bridge::RegisterWorkerRequest);
```

Comment:
Let's move the wrapper dudes into their own module

Reply:
Done